operations/admin_all_configs.go

```diff
@@ -88,5 +97,25 @@ func fetchAndWriteConfig(c *legacyClient, project string) error {
 		return errors.Wrapf(err, "failed to write configuration for project %s", project)
 	}
 
+	if validate {
+		validationErrs, err := ac.ValidateLocalConfig(data)
+		if err != nil {
+			return errors.Wrap(err, "error validating config")
+		}
+
+		if len(validationErrs) == 0 {
+			grip.Infof("%s has no validation errors", project)
+		} else {
+			grip.Infof("validation errors for %s:\n", project)
+			for _, err := range validationErrs {
```

Comment:
This requires the user to scroll up to find validation errors. It would be better to return these to the caller so that they are printed when the catcher is resolved.

Reply:
I don't think these should be returning errors to the caller, since those are assumed to be errors executing the command. Would it be sufficient to not print anything for projects that don't have errors?